notebooks/geo/BepiColombo_FlybyVisualization.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        #Monitor memory
</pre><p>I think you can remove the memory monitoring (after you added the del statement that I suggested above) from the code we want to publish. That was more a debugging step.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>